bmaptool/BmapCreate.py

```diff
@@ -48,8 +48,8 @@
 # pylint: disable=R0902,R0903
 
 import hashlib
-from bmaptools.BmapHelpers import human_size
-from bmaptools import Filemap
+from bmaptool.BmapHelpers import human_size
+from . import Filemap
```

Comment:
In later commits, we have `from bmaptool import <foo>`. Personally I find that more explicit and readable than the "from dot import". However, we might consider using an opinionated formatting tool like `ruff` or `black` and just do whatever it says.

Reply:
Sounds good. I'm not familiar with either of those tools so I'll look into them.

What happened is I wasn't sure how to to the ".BmapHelpers" trick that @JPEWdev mentions later. So I changed one in one way and changed the other in a different way. I'm happy to do the explicit "from bmaptool..." thing, or whatever the formatting tool recommends.